README.md

```diff
@@ -1,6 +1,6 @@
 # Address Book (Level 4)
 
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
+[![Build Status](https://travis-ci.org/CS2103JAN2017-T09-B4/main.svg?branch=master)](https://travis-ci.org/CS2103JAN2017-T09-B4/main.svg?branch=master)
```

Comment:
need to update the other links (codacy, coveralls) to reflect your project not addressbook.